README.md

```diff
@@ -138,20 +61,37 @@ services:
     network_mode: ${DOCKER_MODE}
 ...
 ```
+### Configuring CanDIG modules
+
+Not all CanDIG modules are required for a minimal installation. The `CANDIG_MODULES` and `CANDIG_AUTH_MODULES` define which modules are included in the deployment. 
+
+By default (if you copy the sample file from `etc/env/example.env`) the installation includes the minimal list of modules:
+
+  CANDIG_MODULES=minio htsget-server chord-metadata candig-server candig-data-portal
```

Comment:
We can remove candig-server from the minimal set at this point.

Reply:
I wanted to do that in a separate PR (remove candig-server from the example env and update the docs)